de.metas.async/src/main/sql/postgresql/system/42-de.metas.async/5476160_sys_gh2894_SysConfig_Async_InitDelayMillis.sql

```diff
@@ -0,0 +1,10 @@
+-- 2017-11-02T15:32:55.344
+-- URL zum Konzept
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541168,'S',TO_TIMESTAMP('2017-11-02 15:32:55','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.async','Y','Async_InitDelayMillis',TO_TIMESTAMP('2017-11-02 15:32:55','YYYY-MM-DD HH24:MI:SS'),100,'60000')
```

Comment:
please add some documentation like "When the async component is loaded, this delay makes sure that it was fully started by the time the async component starts to process packages" ...or something more sensible..

Reply:
Thanks. Documentation added in both the description of the sys config and in the code where it is used.